generated/nidaqmx/constants.py

```diff
@@ -765,9 +783,9 @@ class StrainGageRosetteMeasurementType(Enum):
 
 
 class StrainGageRosetteType(Enum):
-    RECTANGULAR = 15968  #: A rectangular rosette consists of three strain gages, each separated by a 45 degree angle.
-    DELTA = 15969  #: A delta rosette consists of three strain gages, each separated by a 60 degree angle.
-    TEE = 15970  #: A tee rosette consists of two gages oriented at 90 degrees with respect to each other.
+    RECTANGULAR_ROSETTE = 15968  #: A rectangular rosette consists of three strain gages, each separated by a 45 degree angle.
+    DELTA_ROSETTE = 15969  #: A delta rosette consists of three strain gages, each separated by a 60 degree angle.
+    TEE_ROSETTE = 15970  #: A tee rosette consists of two gages oriented at 90 degrees with respect to each other.
```

Comment:
Old seems preferable since its less repetition. Rosette is in the enum name.

Reply:
Resolved.